frame/crowdloan-rewards/src/tests.rs

```diff
@@ -95,6 +95,15 @@ fn test_initialize_ok() {
 	});
 }
 
+#[test]
+fn test_initialize_at_ok() {
+	ExtBuilder::default().build().execute_with(|| {
+		assert_ok!(CrowdloanRewards::initialize_at(Origin::root(), 10));
+		assert_eq!(CrowdloanRewards::total_rewards(), 0);
+		assert_eq!(CrowdloanRewards::claimed_rewards(), 0);
+	});
+}
+
```

Comment:
Please add a test to ensure you cannot claim if you initialize_at a future block

Reply:
yessir